pkgs/applications/science/biology/flywheel-cli/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+}:
+
+let
+  inherit (stdenv.targetPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
+in
+stdenv.mkDerivation rec {
+  pname = "flywheel-cli";
+  version = "16.2.0";
+
+  os = {
+    x86_64-darwin = "darwin";
+    x86_64-linux = "linux";
+  }.${system} or throwSystem;
+
+  sha256 = {
+    x86_64-darwin = "sha256-OIyEu3Hsobui9s5+T9nC10SxMw0MhgmTA4SN9Ridyzo=";
+    x86_64-linux = "sha256-SxBjRd95hoh2zwX6IDnkZnTWVduQafPHvnWw8qTuM78=";
+  }.${system} or throwSystem;
+
+  src = fetchurl {
+    url = "https://storage.googleapis.com/flywheel-dist/cli/${version}/fw-${os}_amd64-${version}.zip";
+    inherit sha256;
+  };
```

Comment:
I tried to compile the Flywheel CLI from scratch, utilizing the `Makefile` included with the project. I ran into a bunch of issues (e.g., the project depends on an EOL version of Go, it's a Go + Python project), spent a day trying to get things to work, and decided to take the path of least resistance by using `fetchurl` on their prebuilt binary.

The consequence is we're limited to `x86_64-darwin` and `x86-64-linux`.